src/PageContent/Calendar/Dialog.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './Dialog.css';
+import Header from './Header';
+import DialogContent from './DialogContent';
+
+class Dialog extends Component {
+    render() {
+        return (
+            <dialog className="event-dialog" open>
+                <Header
+                    eventType={this.props.eventType}
```

Comment:
see comment in `PageContainer` line 91. It'll need to be `this.props.delivery.eventType` here, same with the rest.

Reply:
I had to move the delivery set state into component will mount for the state to ready when event card is rendered